processor_function/README.md

```diff
@@ -0,0 +1,185 @@
+# Processor function
+
+<!-- markdownlint-disable-next-line MD013 -->
+The processor function (sometimes referred to as resource or deployment processor) uses [Azure Container Instances (ACI)](https://docs.microsoft.com/en-us/azure/container-instances/) together with the [CNAB container image](../CNAB_container/Dockerfile) based on [Azure CNAB Driver](https://github.com/deislabs/cnab-azure-driver) to execute workspace and workspace service deployments. The workspaces and workspace services are [Porter](https://porter.sh/) bundles.
```

Comment:
perhaps mention something about the responsibility

Reply:
Added some more information. We can revisit this in #431